src/solvers/dgmulti/flux_differencing_gauss_sbp.jl

```diff
@@ -92,6 +92,19 @@ function create_cache(mesh::VertexMappedMesh, equations,
   inv_gauss_weights = inv.(rd.wq)
   Pf = diagm(inv_gauss_weights) * interp_matrix_gauss_to_face'
 
+  # Conditionally use sparse operators if they can be estimated to be faster than dense operators
+  # (based on some benchmarks on an AMD Ryzen Threadripper 3990X 64-Core Processor).
+  # 2D operators are mostly not sparse enough and too small to benefit from sparse representations.
+  # TODO: Check whether SuiteSparseGraphBLAS.jl can be used to get even better performance
```

Comment:
```suggestion
  # TODO: DGMulti. Check whether SuiteSparseGraphBLAS.jl can be used to get even better performance
```
Just for labeling and search later.